config/lib/express.js

```diff
@@ -40,8 +40,9 @@ module.exports.initLocalVariables = function (app) {
 
   // Passing the request url to environment locals
   app.use(function (req, res, next) {
-    res.locals.host = req.protocol + '://' + req.hostname;
-    res.locals.url = req.protocol + '://' + req.headers.host + req.originalUrl;
+    var domain = config.app.domain || req.headers.hostname;
```

Comment:
@mleanos Wait, no. Both `req.hostname` and `req.headers.hostname` won't work since they both exclude the port. The values in `res.locals` should include the port (if its not 80), right?

So we have to use `req.headers.host` which includes the port.